src/components/App.js

```diff
@@ -213,7 +211,26 @@ const App = () => {
                             ],
                     },
                 })
-
+                /* VERSION-TOGGLE: remove if condition when 42 is lowest supported version,
+                 * because by then metaData and parentGraphMap are always returned by the
+                 * web API. */
+                if (
+                    data.eventVisualization.metaData &&
+                    data.eventVisualization.parentGraphMap
+                ) {
+                    dispatch(
+                        acAddMetadata(
+                            transformMetaDataResponseObject(
+                                data.eventVisualization.metaData
+                            )
+                        )
+                    )
+                    dispatch(
+                        acAddParentGraphMap(
+                            data.eventVisualization.parentGraphMap
+                        )
+                    )
+                }
```

Comment:
@jenniferarnesen I am updating the redux store conditionally here, because a change in the `metaData` state causes pretty much the full app to re-render. And `acAddMetadata` will produce a new `metaData` object even if we were to pass it `{}` or `undefined`. Since these properties don't exist pre-v42, I opted for an `if` here.

Even though this code doesn't technically branch on the version, I opted to add the `VERSION-TOGGLE` comment because this can be simplified in the future. I guess I am not strictly speaking the guidelines I outlined in the [FAQ](https://docs.google.com/document/d/1SnlYnXxWPps9y1jHwvAkaldokXVCWNIPFIFobLIoJs4/edit?tab=t.0#heading=h.508b6hssylfs), but this seemed like the most pragmatic solution for the current situation. Let me know if it makes sense to you.